wgpu/src/backend/webgpu/webgpu_sys/gen_Gpu.rs

```diff
@@ -0,0 +1,73 @@
+// DO NOT EDIT THIS FILE!
```

Comment:
can you generate a .gitattributes file like this https://github.com/rerun-io/rerun/blob/main/crates/re_types_core/.gitattributes with `linguist-generated=true` for each file? Then the files show up as generated in reviews :)

Reply:
Oh cool! How do you even find out about this stuff :)

Reply:
@cwfitzgerald: I imagine @Wumpf read these GH docs: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
I wonder if this is also an opportunity to mark our vendored code, too, with this? https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#vendored-code

Reply:
@ErichDonGubler what vendored code were you thinking of?